src/lib.rs

```diff
@@ -157,19 +165,15 @@ impl Bloom {
     }
 }
 
-/// Returns 2 hashes for the given item
 fn double_hashing_hashes<T: Hash>(item: T) -> (u64, u64) {
-    // Using seahash v4 as a portable hasher
-    let mut hasher = seahash::SeaHasher::with_seeds(
-        0x16f11fe89b0d677c,
-        0xb480a793d8e6c86c,
-        0x6fe2e5aaf078ebc9,
-        0x14f994a4c5259381,
-    );
+    // Using xxh3-64 with default seed/secret as a portable hasher.
+    let mut hasher = xxhash_rust::xxh3::Xxh3::new();
     item.hash(&mut hasher);
     let h1 = hasher.finish();
 
-    // write a nul byte to the existing state and get another hash
+    // Write a nul byte to the existing state and get another hash.
+    // This is appropriate when using a very high quality hasher,
+    // which we know is the case.
     0u8.hash(&mut hasher);
     // h2 hash shouldn't be 0 for double hashing
     let h2 = hasher.finish().max(1);
```

Comment:
Could we leave a comment explaining why the first one doesn't need `.max(1)`?

Reply:
I don't know how to word that. It's easy to comment when something is a special (h1) but not obvious otherwise. The linked Double hashing article should be enough honestly.

Reply:
Sounds good :+1: